ui-tests/src/test/java/org/openmrs/reference/page/VisitNotePage.java

```diff
@@ -76,19 +89,36 @@ public void enterSecondaryDiagnosis(String diag) {
     }
 
     public void deleteDiagnosis() {
-        clickOn(DEL_DIAGNOSIS);
+        clickOn(DELETE_DIAGNOSIS);
     }
-
+    
+    public void confirmDeleteDiagnosis() {
+        clickOn(CONFORM_DELETE_DIAGNOSIS);
+    }
+    
+    public void cancelDiagnosisButton() {
+        clickOn(CANCEL_DELETE_DIAGNOSIS);
+    }
+    
+    public void deleteEncounterNote() {
+        findElement(DELETE_ENCOUNTER_NOTE).click();
+    }
+    
+    public void confirmDeleteButton() {
+        findElement(COMFIRM_DELETE_BUTTON).click();
+    }
+  
     public String primaryDiagnosis() {
         return findElement(PRIMARY_DIAGNOSIS_ELEMENT).getText().trim();
     }
 
     public String secondaryDiagnosis() {
-        return findElement(SECONDARY_DIAGNOSIS_ELEMENT).getText();
+        return findElement(SECONDARY_DIAGNOSIS_ELEMENT).getText().trim();
     }
 
     public ClinicianFacingPatientDashboardPage save() {
         clickOn(SAVE_VISIT_NOTE);
         return new ClinicianFacingPatientDashboardPage(this);
     }
 }
+
```

Comment:
`LN-124` should we not remove this empty line? I can see already another empty line existing to cater for contributors git collaboration. If `LN-124` remains we will have two last empty lines which may not be a good thing!

Reply:
> I can see already another empty line existing`(behind the seen)` t that caters for contributors git collaboration.
Which line is that, this line 124 is the last line, btw what do you mean by behind scenes ,Does git provides this behind scenes scenarios, there is no line 125. it only stops on line 124

Reply:
> btw what do you mean by behind scenes ,Does git provides this behind scenes scenarios

I often use the git functionality of Edit file to observe the empty line!

![Screenshot from 2021-08-04 10-49-20](https://user-images.githubusercontent.com/38831673/128143495-ece6d400-98b8-455b-a277-56a0bb272a8e.png)